constraints/miden-vm/bitwise.air

```diff
@@ -70,10 +77,28 @@ ev output_aggregation(main: [s, a, b, a_limb[4], b_limb[4], zp, z]):
         z = zp * 16 + a_xor_b when s
         z = zp * 16 + a_and_b when !s
 
+
+# Enforces the constraints for bitwise chiplet bus.
+ev bus_constraints([a, b, z], [b_chip]):
```

Comment:
A few small nits here:
1. We don't use "constraints" in the other names, so I'd probably prefer to just call this `bus` or `bitwise_chiplet_bus`
2. There's actually only one constraint, so the comment can say "Enforces the constraint for the bitwise chiplet bus"
3. Let's include the degree of the constraint in this comment as well.